noxfile.py

```diff
@@ -220,6 +234,26 @@ def showcase_library(
                 external=True,
                 silent=True,
             )
+            # TODO: The below section updates the showcase service yaml
+            # to test experimental async rest transport. It must be
+            # removed once support for async rest is GA.
+            # See related issue: https://github.com/googleapis/gapic-generator-python/issues/2121.
+            if rest_async_io_enabled:
+                # Install pyYAML for yaml
+                session.install("pyYAML")
+
+                python_settings = [
+                    {
+                        'version': 'google.showcase.v1beta1',
+                        'python_settings': {
+                            'experimental_features': {
+                                'rest_async_io_enabled': True
+                            }
+                        }
+                    }
+                ]
+                update_service_yaml = _add_python_settings(tmp_dir, python_settings)
+                session.run("python", "-c" f"{update_service_yaml}")
```

Comment:
What do these lines do? You modify the service YAML by calling the function which returns `None`...and then your'e calling `python -c None` ? Could you clarify?

Reply:
- `session.run` runs the code within `_add_python_settings` inside the nox session environment.
- `_add_python_settings` modifies the service yaml config within the temporary directory and add the configuration to enable the optional feature. The configuration is defined in `python_settings`, right above this line.

Reply:
Oh, got it. Thanks!